src/components/grouped-select/country-population-data.ts

```diff
@@ -0,0 +1,222 @@
+export const countryData =[
```

Comment:
use from `generated-data/country-state.json`

Reply:
Removed this